### ISSUE TITLE
Can't get it to run

### ISSUE THREAD

**Issue by geleeroyale** [rauljordan/daily-discord-summarizer#1]
However I try to run this - I encounter bugs. They all have to do with the database. How should we prepare the db exactly? I tried an empty one, running migrations from the migrations directory.

But I still encounter errors. Is this maintained? Thank you!

**Comment by alibama** [comment]
yeah - same here = can't get anything to run.... 

"cargo install sqlx-cli" solved some issues, but didn't get the job done

**Comment by alibama** [comment]
also ran the following sql in to the db to see if that would help 

https://github.com/rauljordan/daily-discord-summarizer/blob/main/migrations/20231201181151_initial_migration.sql

and that didn't really do anything either


-- Create the 'daily_digests' table
CREATE TABLE daily_digests (
    id INTEGER PRIMARY KEY AUTOINCREMENT,
    text TEXT NOT NULL,
    timestamp DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP
);

-- Create the 'summaries' table with a foreign key reference to 'daily_digests'
CREATE TABLE summaries (
    id INTEGER PRIMARY KEY AUTOINCREMENT,
    daily_digest_id INTEGER,
    text TEXT NOT NULL,
    timestamp DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
    FOREIGN KEY (daily_digest_id) REFERENCES daily_digests(id)
);

**Comment by mrjcleaver** [comment]
Did you find an alternative?

**Comment by alibama** [comment]
yeah https://github.com/Tyrrrz/DiscordChatExporter worked fine for what i need
